contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1510,6 +1523,150 @@ modify_insert_stmt(InsertStmt *stmt, Oid relid)
 
 }
 
+/*
+ * Stores view object's TSQL definition to bbf_view_def catalog
+ */
+static void
+pltsql_store_view_definition(const char *queryString, ObjectAddress address)
```

Comment:
What happens when we run into error during this function? Will the view be created?

Reply:
View won't be created and error will be thrown.